BrickController2/BrickController2/UI/Pages/ControllerTesterPage.xaml

```diff
@@ -27,6 +27,10 @@
                     <CollectionView.ItemTemplate>
                         <DataTemplate>
                             <StackLayout Orientation="Vertical" Padding="10">
+                                <StackLayout Orientation="Horizontal">
```

Comment:
Nit pick: this add yet another line, could you consider adding it as an column? So as more events can be shown on single screen.

Reply:
I think this is a good solution.
I added a seperator between ControllerDeviceId and EventCode.

Reply:
...strange: die seperating space (last char in) in <Label Text="ControllerId: " FontSize="Medium"/> is ignored in the WinApp - on Android it's shown...?